### DISCUSSION TITLE
Strings

### DISCUSSION THREAD

**Discussion by igutiez** [frangoteam/FUXA#317] [Ideas]
I am Siemens PLC user.

Could be implemented strings as variable type using SiemensS7? Because it is needed many times for recipe names, parts names etc that are needed to be displayed in HMI/SCADA.

BR

**Comment by unocelli** [comment]
Hi,
I will investigate...

**Reply by igutiez** [reply]
Ok, thanks :)

**Comment by unocelli** [comment]
I have a question, how you configure a char (string) db.xy ?

**Reply by igutiez** [reply]
In node-red, or OPC-DA, the link is something similar to DB1,S0.28 where DB1 y the DB1, S means that is an string, 0 is the starting byte and 28 the amount of characters of the DB. The amount of readed bytes is 30 starting at byte 0, and the firts char is the byte 2 because the byte 0 is the string lenght (28) and the byte 1 is the real amount of the string, for example 5 in case of "hello".
